Students/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Newtonsoft.Json;
+using System.Linq;
+
+namespace Students
```

Comment:
checkout [this lisk](https://stackoverflow.com/questions/360643/is-it-a-bad-practice-to-have-multiple-classes-in-the-same-file), better to have a single class file for each class